test/emailverification/emailverification.claim.test.js

```diff
@@ -0,0 +1,143 @@
+/* Copyright (c) 2024, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+const { printPath, setupST, killAllST, cleanST, startST: globalStartST, createTenant } = require("../utils");
+let assert = require("assert");
+const { recipesMock, randomString } = require("../../api-mock");
+const { EmailPassword, EmailVerification, Session, supertokens } = recipesMock;
```

Comment:
Please merge the latest `3.0` branch into this, sadly this test approach will no longer work. This change was required, because it wasn't actually testing the `impl` in the backend SDK we are supposed to be testing, only the impl defined in the dependencies of this project.

Reply:
To adapt these test, please use `createNewSessionWithoutRequestResponse` and just call `assertClaims` on the session object instead

Reply:
done